js/jsmind.js

```diff
@@ -2800,6 +2806,40 @@
                 this.graph.draw_line(pout, pin, _offset);
             }
         },
+
+        // Drag the whole mind map with your mouse, when it's larger that the container
+        _drag_nodes: function () {
+            if (this.opts.draggable) {
+                // Dragging disabled by default.
+                let dragging = false
+                let x, y
+                if (this.opts.hide_scrollbars) {
```

Comment:
I guess the `hide_scrollbars` won't take effect when `draggable=false`.
If we mean it, `hide_scrollbars` should be renamed to `hide_scrollbars_when_draggable`
what do you think?